doc/array.md

```diff
@@ -35,6 +35,13 @@ To create an array of size `n` in which all the elements are initialized to some
 -- #['a', 'a', 'a', 'a', 'a']
 ```
 
+To create an array from a range of numbers use `List.range` and convert the result to an Array:
```

Comment:
This is not a good example. We should eventually have `Array.range` in the stdlib which avoids the overhead of creating a temporary list.

Reply:
When that happens we can update this comment.  But right now folks will puzzle on how to do it and this is the best way for now.